tensorflow/python/framework/graph_io.py

```diff
@@ -58,6 +60,14 @@ def write_graph(graph_or_graph_def, logdir, name, as_text=True):
   else:
     graph_def = graph_or_graph_def
 
+  if sys.byteorder == 'big':
+    if hasattr(graph_def, 'node'):
```

Comment:
Is this condition guaranteed only for frozen graph?

Reply:
No, having attribute `node` can't guarantee it is a frozen graph. I've refactored the `swap_tensor_content_*` function names accordingly. Thanks!